src/CatCard/CatCard.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Image,
+  Text,
+  View,
+} from 'react-native';
+
+import { styles } from './styles';
+
+export const CatCard = ({ cat }) => (
+  <View style={styles.catCard}>
+    <View style={styles.headInfo}>
+      <Text style={styles.name}>{cat.name}</Text>
+      <Text style={styles.breed}>{cat.breed}</Text>
+    </View>
+    <View style={styles.divider} />
+    <Image source={require('../catImg/angora.png')} style={styles.img} />
```

Comment:
Let's make image dynamic :)
As an option you can add link to the image from such services like unsplash.com because in real life when you fetch data from server you receive link to image and do not store it locally